parm/wflow/da_data_preproc.yaml

```diff
@@ -0,0 +1,88 @@
+# This group contains all the tasks needed for preprocessing tasks for
+# RRFS DA.
+
+default_data_preproc_task: &default_preproc
+  account: '&ACCOUNT;'
+  attrs:
+    cycledefs: #cycle_type#
+    maxtries: '1'
+  envars: &default_envars
+    GLOBAL_VAR_DEFNS_FP: '&GLOBAL_VAR_DEFNS_FP;'
+    USHdir: '&USHdir;'
+    PDY: !cycstr "@Y@m@d"
+    cyc: !cycstr "@H"
+    subcyc: !cycstr "@M"
+    LOGDIR: !cycstr "&LOGDIR;"
+    CYCLE_TYPE: '#cycle_type#'
+  native: '{{ platform.SCHED_NATIVE_CMD }}'
+  nodes: '{{ nnodes }}:ppn={{ ppn }}'
+  nnodes: 1
+  nodesize: "&NCORES_PER_NODE;"
+  ppn: 1
+  partition: '{% if platform.get("PARTITION_DEFAULT") %}&PARTITION_DEFAULT;{% else %}None{% endif %}'
+  queue: '&QUEUE_DEFAULT;'
+  walltime: 00:25:00
+
+metatask_process_obs_cycle_type:
+  var:
+    cycledefs_type: prodcyc,prodcyc_long
+    cycle_type: prod
+
+  task_process_radarref_#cycle_type#:
+    <<: *default_preproc
+    command: '&LOAD_MODULES_RUN_TASK_FP; "process_obs" "&JOBSdir;/JREGIONAL_PROCESS_RADARREF"'
+    ppn: 24
+    dependency:
+      or:
+        taskdep:
+          attrs:
+            task: get_da_obs_radar
```

Comment:
These names are placeholders for what comes of the retrieve data efforts underway by @mkavulich.

I am leaving the option here to either have this task run, or to have the script know where to look on disk for the sake of efficiency in the real-time runs.